pytorch_lightning/callbacks/base.py

```diff
@@ -248,10 +248,22 @@ def on_train_end(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -
         """Called when the train ends."""
 
     def on_pretrain_routine_start(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -> None:
-        """Called when the pretrain routine begins."""
+        r"""
+        .. deprecated:: v1.6
+            This callback hook was deprecated in v1.6 and will be removed in v1.8. Use
+            ``on_fit_start`` instead.
+
+        Called when the pretrain routine begins.
+        """
 
     def on_pretrain_routine_end(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -> None:
-        """Called when the pretrain routine ends."""
+        r"""
+        .. deprecated:: v1.6
+            This callback hook was deprecated in v1.6 and will be removed in v1.8. Use
+            ``on_fit_start`` instead.
```

Comment:
```suggestion
            ``on_fit_end`` instead.
```

Reply:
@daniellepintz On fit end is not correct. That happens at the very end of the call

Reply:
Oh, sorry, I was confused.